template/nuxt-app/assets/definitions.styl

```
@@ -44,6 +48,13 @@ spacing-s = 30px
 spacing-s-min = spacing-s * 0.8
 spacing-xs = 15px
 spacing-xs-min = spacing-xs * 0.9
+spacing-xxs = 8px
+spacing-xxs-min = spacing-xxs * 0.9
+
+
+// Misc
+radius = 5px
+base-speed = 200ms
```

Trying to organize these in buckets

```suggestion
// UI
card-radius = 5px

// Animation
base-speed = 200ms
```